components/brave_wallet/browser/internal/hd_key_ed25519.h

```diff
@@ -7,48 +7,63 @@
 #define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_ED25519_H_
 
 #include <memory>
+#include <optional>
 #include <string>
-#include <vector>
 
 #include "base/containers/span.h"
-#include "brave/components/brave_wallet/rust/lib.rs.h"
+#include "base/gtest_prod_util.h"
+#include "brave/components/brave_wallet/browser/internal/hd_key_utils.h"
 
 namespace brave_wallet {
 
-// This class implement basic EdDSA over ed25519 functionality of bip32-ed25519
+inline constexpr size_t kEd25519PrivateKeySize = 32;
```

Comment:
`ED25519_PRIVATE_KEY_LEN`, `ED25519_PUBLIC_KEY_LEN`
also it looks like `kEd25519PrivateKeySize` is incorrect and should be 64?

Reply:
32 bytes is private key size per spec. 

64 bytes is what OpenSSL names as private_key which is actually a pair of private and public keys.

Reply:
I had left a comment elsewhere about this, but I guess both me and Brian were confused, so I guess maybe we need some better naming, the comment above could also better explain this distinction, besides the link to the rfc.